tests/acceptance/run.sh

```diff
@@ -524,15 +524,21 @@ then
 	DIR_URL="${TESTING_APP_URL}dir"
 	# test that server is up and running, and testing app is enabled.
 	assert_server_up ${TEST_SERVER_URL}
-	assert_testing_app_enabled ${TEST_SERVER_URL}
+	if [ "${TEST_OCIS}" == "false" ]
```

Comment:
```suggestion
	if [ "${TEST_OCIS}" != "true" ]
```

I suspect that this will never happen - `TEST_OCIS` will never be set to the string "false", so `assert_testing_app_enabled` will never be done any more.